app/src/main/java/com/android/phuston/noted/NoteFragment.java

```diff
@@ -0,0 +1,138 @@
+package com.android.phuston.noted;
+
+import android.app.AlertDialog;
+import android.app.Fragment;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.AdapterView;
+import android.widget.EditText;
+import android.widget.ListView;
+
+import com.melnykov.fab.FloatingActionButton;
+
+import java.util.ArrayList;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+
+/**
+ * Created by patrick on 9/5/15.
+ */
+public class NoteFragment extends Fragment {
+
+    @Bind(R.id.listview_notes) ListView listview;
+    @Bind(R.id.fab) FloatingActionButton fab;
+
+    private NoteAdapter mNotesAdapter;
+    private ArrayList<String> mNotes;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState){
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(final LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+
+        mNotesAdapter = new NoteAdapter(mNotes, getActivity());
+
+        View rootView = inflater.inflate(R.layout.fragment_note, container, false);
+        ButterKnife.bind(this, rootView);
+
+        listview.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                final int pos = position;
+
+                AlertDialog.Builder alertBuilder = new AlertDialog.Builder(getActivity());
+
+                alertBuilder.setTitle("Edit Note");
```

Comment:
option + enter on string literals in android studio will give you the option to quickly extract to a string resource